src/axom/quest/examples/quest_marching_cubes_example.cpp

```diff
@@ -176,18 +204,20 @@ struct Input
     ndim = std::max(ndim, fcnCenter.size());
     ndim = std::max(ndim, inPlane.size());
     ndim = std::max(ndim, perpDir.size());
+    ndim = std::max(ndim, gyroidScale.size());
     SLIC_ASSERT_MSG((fcnCenter.empty() || fcnCenter.size() == ndim) &&
                       (inPlane.empty() || inPlane.size() == ndim) &&
-                      (perpDir.empty() || perpDir.size() == ndim),
+                      (perpDir.empty() || perpDir.size() == ndim) &&
+                      (gyroidScale.empty() || gyroidScale.size() == ndim),
                     "fcnCenter, inPlane and perpDir must have consistent sizes "
                     "if specified.");
 
     usingPlanar = !perpDir.empty();
     usingRound = !fcnCenter.empty();
-    SLIC_ASSERT_MSG(
-      usingPlanar || usingRound,
-      "You must specify a planar scalar function or a round scalar"
-      " function or both.");
+    usingGyroid = !gyroidScale.empty();
+    SLIC_ASSERT_MSG(usingPlanar || usingRound || usingGyroid,
+                    "No functions specified.  Please specify a comibnation of "
+                    "round, gyroid or planar functions.");
```

Comment:
I think this check could be enforced by CLI11

Reply:
Thanks for the suggestion.  Used it.